### PR TITLE
Iss435: Add polygon & multipolygon support for geoJsonReader.

### PR DESCRIPTION
**Pull request by dcjohnston**
I've updated the geojson reader example to include both a polygon of arizona and a multipolygon of california. Both are being rendered properly on the map using the 'polygon' feature class. As far as style goes, I tried to keep my changes minimal, with most changes occuring in _addFeature and read methods of the jsonReader class.

### CODE REVIEW

**Comment by aashish24** [comment]
@dcjohnston this is really cool. I am thinking we should start manual documentation on some of these. We have raedthedocs but I don't think its that user friendly. Should we start with Github wiki for now? What you think @jbeezley

File: sources.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
         "geomFeature.js",
         "graphFeature.js",
         "contourFeature.js",
+	"choroplethFeature.js",
```
Comment (jbeezley) [inline sources.json]: This is already included.

File: sources.json [hunk 2]
```diff
@@ -46,7 +47,8 @@
         "init.js",
         "wigglemaps.js",
         "leaflet.js",
-        "clustering.js"
+        "clustering.js",
+	"scales.js"
```
Comment (jbeezley) [inline sources.json]: Could you satiate my OCD and fix the whitespace here?

**Comment by jbeezley** [comment]
There are some whitespace only changes in `jsonReader.js` that I would prefer reverted as well as some legitimate jscs/jshint errors.  Other than those small things, this is great.

**Comment by jbeezley** [comment]
Go ahead and merge it.

**Comment by aashish24** [comment]
I want that number plate :+1:

**Comment by danlamanna** [comment]
This will fix #435

**Pull request merged**